common/lib/xmodule/xmodule/mako_module.py

```diff
@@ -9,9 +9,15 @@
 
 
 class MakoDescriptorSystem(DescriptorSystem):  # lint-amnesty, pylint: disable=abstract-method
+    """
+    Descriptor system that renders mako templates.
+    """
     def __init__(self, render_template, **kwargs):
         super().__init__(**kwargs)
 
+        # Import is placed here to avoid model import at project startup.
+        from common.djangoapps.edxmako.services import MakoService
+        self._services['mako'] = MakoService()
```

Comment:
I started to do that, but instead, moved this hack to the Instructor Dashboard where it's using the HtmlBlock, which [provoked this change in the first place](https://github.com/edx/edx-platform/pull/28571/commits/96d92083748145aa41001d85a28f293b338239ff).

Reply:
See https://github.com/open-craft/edx-platform/blob/jill/bd-13-render_template/lms/djangoapps/instructor/views/instructor_dashboard.py#L645-L666

Reply:
Ugh.. removing this caused other test failures:

* https://build.testeng.edx.org/job/edx-platform-python-pipeline-pr/35104/testReport/junit/cms.djangoapps.contentstore.tests.test_contentstore/MiscCourseTests/Run_Tests___cms_unit___test_module_preview_in_whitelist/
* https://build.testeng.edx.org/job/edx-platform-python-pipeline-pr/35104/testReport/junit/xmodule.xmodule.tests.test_library_root/TestLibraryRoot/Run_Tests___commonlib_unit___test_library_author_view/
* https://build.testeng.edx.org/job/edx-platform-python-pipeline-pr/35104/testReport/junit/xmodule.xmodule.tests.test_library_root/TestLibraryRoot/Run_Tests___commonlib_unit___test_library_author_view/

So I'm inclined to leave it in place, with a comment to explain why.

When we factor out MakoDescriptorSystem, then it can go away.